### DISCUSSION TITLE
unlabeled data how will effect on my model

### DISCUSSION THREAD

**Discussion by hasanberatsoke-cy** [ultralytics/yolov5#12242] [Q&A] [answered]
Hello,
If I add unlabeled data, unrelated to the object I want to detect, to the yolov5 or yolov7 dataset, will it have a positive effect on my model

So, if I add some random images from imagenet (without labels) to a face-detecting Yolo model, will I strengthen my model?

**Comment by danielzhangau** [comment, marked as answer]
The [Ultralytics YOLOv5 documentation](https://docs.ultralytics.com/yolov5/tutorials/tips_for_best_training_results/) mentions the use of "background images" specifically to reduce false positives (FPs) during object detection. These are images that contain no instances of the objects you're interested in detecting and are included in your training set without any labels. According to the guide, they recommend using about 0-10% background images relative to your dataset size to help reduce FPs.

In this context, adding unlabeled, unrelated images would act as these "background images." The key purpose is to teach the model what doesn't constitute the object of interest, thereby reducing the likelihood of false positives. This is directly in line with what you're asking; adding such images should improve the model's ability to discern the object you're interested in detecting from other unrelated objects or backgrounds.

However, the addition should be done carefully to maintain the balance in your dataset and not to introduce any class imbalance or skewed learning.

So, according to this specific guide, adding random images from ImageNet without labels (effectively treating them as background images) is a recommended practice to improve your model's performance by reducing false positives.

**Reply by hasanberatsoke-cy** [reply]
@danielzhangau Thank you so much.
